src/core/components/accordion/index.tsx

```diff
@@ -1,27 +1,105 @@
-import React, { ReactNode } from "react"
-import { accordion, label } from "./styles"
+import React, { useState, ReactNode } from "react"
+import {
+	accordion,
+	labelText,
+	titleRow,
+	accordionRow,
+	accordionToggle,
+	showAccordionElement,
+	hideAccordionElement,
+	grey,
+	white,
+} from "./styles"
 import { Props } from "@guardian/src-helpers"
+import { SvgChevronDownSingle, SvgChevronUpSingle } from "@guardian/src-svgs"
 
-interface AccordionProps extends Props {
-	children: ReactNode
-}
+type Colours = "grey" | "white"
 
-const Accordion = ({ children }: AccordionProps) => {
-	return <div css={accordion}>{children}</div>
+const backgroundColours = {
+	grey: grey,
+	white: white,
```

Comment:
I wonder if we should just support one of these for now to reduce complexity and API surface area? Maybe just the grey background if it serves your purpose?

Reply:
Do you mean that I should just make the background grey and remove this from the API?

Reply:
(I have gone ahead and done that, since keeping the API seems YAGNI with only one colour)

Reply:
Yes, a transparent background seems like a good solution. I have done that.